applications/MappingApplication/custom_utilities/interface_object.h

```diff
@@ -110,20 +110,9 @@ class InterfaceObject : public Point<3>
 
     bool IsInBoundingBox(double* pBoundingBox[])
     {
-        // xmax, xmin,  ymax, ymin,  zmax, zmin
-        bool is_inside = false;
+        std::vector<double> point_to_check {this->X(), this->Y(), this->Z()};
 
-        if (this->X() < *pBoundingBox[0] && this->X() > *pBoundingBox[1])   // check x-direction
-        {
-            if (this->Y() < *pBoundingBox[2] && this->Y() > *pBoundingBox[3])   // check y-direction
-            {
-                if (this->Z() < *pBoundingBox[4] && this->Z() > *pBoundingBox[5])   // check z-direction
-                {
-                    is_inside = true;
-                }
-            }
-        }
-        return is_inside;
+        return MapperUtilities::PointIsInsideBoundingBox(*pBoundingBox, point_to_check);;
```

Comment:
I know, but in other cases I don't have an underlying point. e.g. [here](https://github.com/KratosMultiphysics/Kratos/blob/feature_mappingApp_improve_useability/applications/MappingApplication/custom_utilities/mapper_utilities.h#L313), should I create a Point and pass it to the function (and change the function to accept only Points?) Or is it better to overload the function.
I would go for creating a Point, I think this shouldn't be too expensive?